rldb.py

```diff
@@ -0,0 +1,100 @@
+from os import path
+import sqlite3
+
+# Original Repository: https://github.com/eibex/reaction-light
+# License: MIT - Copyright 2019-2020 eibex
+
+directory = path.dirname(path.realpath(__file__))
+database = sqlite3.connect(f"{directory}/files/reactionlight.db")
+db = database.cursor()
+
+db.execute("CREATE TABLE IF NOT EXISTS 'channels' ('message_id' INT, 'channel' INT);")
+
+embeds_creation = {}
+
+class EmbedCreationTracker:
+    def __init__(self, user, channel):
+        self.user = user
+        self.channel = channel
+        self.step = 1
+        self.target_channel = None
+        self.combos = {}
+        self.message_id = None
+
+    def commit(self):
+        db.execute(f"CREATE TABLE '{self.message_id}' ('reaction' NVCARCHAR, 'role_id' INT);")
+        db.execute(f"INSERT INTO 'channels' ('message_id', 'channel') values('{self.message_id}', '{self.target_channel}');")
```

Comment:
Creating a table for each message seems way too overkill for me. Why not creating a single table for every messages, and another table for the reaction/roles with a foreign key linking them to their message instead ?

Reply:
So, I should re-implement what `id.csv` is doing with foreign IDs?

Reply:
I think I can do this with only two tables instead of 3 actually. I will push some changes soon.